src/libs/firebase/firestore/featureFlags/newIdentificationFlow.native.test.ts

```diff
@@ -0,0 +1,56 @@
+import { useEnableNewIdentificationFlow } from 'libs/firebase/firestore/featureFlags/newIdentificationFlow'
+import firestore from 'libs/firebase/shims/firestore'
+import { renderHook, waitFor } from 'tests/utils'
+
+import { build } from '../../../../../package.json'
+
+jest.mock('@react-native-firebase/firestore')
+
+const { collection } = firestore()
+
+const mockGet = jest.fn()
+
+describe('useEnableNewIdentificationFlow', () => {
+  beforeAll(() =>
+    collection('newIdentificationFlow')
+      .doc('testing')
+      // @ts-expect-error is a mock
+      .get.mockResolvedValue({
+        get: mockGet,
+      })
+  )
+
+  it('should call the right firestore collection: featureFlags', () => {
+    renderHook(() => useEnableNewIdentificationFlow())
+
+    expect(collection).toHaveBeenCalledWith('featureFlags')
+  })
+
+  it.each`
+    enableNewIdentificationFlow | minimalBuildNumber
+    ${false}                    | ${undefined}
+    ${false}                    | ${build + 1}
+    ${true}                     | ${build}
+    ${true}                     | ${build - 1}
```

Comment:
je recommande de découper en au moins 2 tests

ex:
should be disabled when the current build number is below the feature flag's minimal build number
should be enabled when the current build number is greater or equal to the feature flag's minimal build number

Reply:
ok

Reply:
Ça voudrait dire en faire un 3e juste pour le cas du undefined. Ça m'embête un peu de dupliquer tout le contenu du test 3 fois. J'ajoute un paramètre testTitle au it.each pour expliciter